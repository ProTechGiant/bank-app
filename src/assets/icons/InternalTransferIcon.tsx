import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function InternalTransferIcon({ width = 20, height = 20, color = "#2E2E2E" }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 20 20" fill={color} xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M18.403 14.9997V13.333H6.73633V3.33301H8.403L5.903 0.833008L3.40299 3.33301H5.06966V4.99967H1.73633V6.66634H5.06966V13.333C5.06966 14.2497 5.81966 14.9997 6.73633 14.9997H13.403V16.6663H11.7363L14.2363 19.1663L16.7363 16.6663H15.0697V14.9997H18.403ZM8.403 6.66634H13.403V11.6663H15.0697V6.66634C15.0697 5.74967 14.3197 4.99967 13.403 4.99967H8.403V6.66634Z"
        fill={color}
      />
    </Svg>
  );
}
