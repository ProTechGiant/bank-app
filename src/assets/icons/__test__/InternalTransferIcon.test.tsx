import { create } from "react-test-renderer";

import { InternalTransferIcon } from "../InternalTransferIcon";

describe("icon/InternalTransferIcon", () => {
  it("renders a InternalTransferIcon", () => {
    const result = create(<InternalTransferIcon />);

    expect(result).toMatchSnapshot();
  });
});
