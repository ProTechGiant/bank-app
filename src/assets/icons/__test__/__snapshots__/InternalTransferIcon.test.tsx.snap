// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/InternalTransferIcon renders a InternalTransferIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="20"
  bbWidth="20"
  fill="#2E2E2E"
  focusable={false}
  height={20}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 20,
        "width": 20,
      },
    ]
  }
  vbHeight={20}
  vbWidth={20}
  width={20}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={
      {
        "payload": 4281216558,
        "type": 0,
      }
    }
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M18.403 14.9997V13.333H6.73633V3.33301H8.403L5.903 0.833008L3.40299 3.33301H5.06966V4.99967H1.73633V6.66634H5.06966V13.333C5.06966 14.2497 5.81966 14.9997 6.73633 14.9997H13.403V16.6663H11.7363L14.2363 19.1663L16.7363 16.6663H15.0697V14.9997H18.403ZM8.403 6.66634H13.403V11.6663H15.0697V6.66634C15.0697 5.74967 14.3197 4.99967 13.403 4.99967H8.403V6.66634Z"
      fill={
        {
          "payload": 4281216558,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
